lib/build-tools/near-bindgen-exporter.js

```diff
@@ -4,7 +4,7 @@ export default function () {
         visitor: {
             ClassDeclaration(path) {
                 let classNode = path.node;
-                if (classNode.decorators && classNode.decorators[0].expression.name == 'NearBindgen') {
+                if (classNode.decorators && classNode.decorators[0].expression.callee.name == 'NearBindgen') {
```

Comment:
Not related to this PR, I thought we have set generated `.d.ts` and `.js` diff not showing in github, but seems it's not working

Reply:
Not sure... they were hidden for me.